help_to_heat/portal/download_views.py

```diff
@@ -297,7 +405,89 @@ def add_extra_row_data(referral, exclude_pii=False):
+        "energy_consumption_potential": epc_data.get("energy-consumption-potential"),
+        "co2_emissions_current": epc_data.get("co2-emissions-current"),
+        "co2_emission_current_per_floor_area": epc_data.get("co2-emiss-curr-per-floor-area"),
+        "co2_emissions_potential": epc_data.get("co2-emissions-potential"),
+        "total_floor_area": epc_data.get("total-floor-area"),
+        "energy_tariff": epc_data.get("energy-tariff"),
+        "mains_gas_flag": epc_data.get("mains-gas-flag"),
+        "floor_level": epc_data.get("floor-level"),
+        "flat_top_storey": epc_data.get("flat-top-storey"),
+        "flat_storey_count": epc_data.get("flat-storey-count"),
+        "main_heating_controls": epc_data.get("main-heating-controls"),
+        "multi_glaze_proportion": epc_data.get("multi-glaze-proportion"),
+        "extension_count": epc_data.get("extension-count"),
+        "number_habitable_rooms": epc_data.get("number-habitable-rooms"),
+        "number_heated_rooms": epc_data.get("number-heated-rooms"),
+        "floor_description": epc_data.get("floor-description"),
+        "floor_energy_efficiency": epc_data.get("floor-energy-eff"),
+        "windows_description": epc_data.get("windows-description"),
+        "epc_walls_description": epc_data.get("walls-description"),
+        "walls_energy_efficiency": epc_data.get("walls-energy-eff"),
+        "walls_environmental_efficiency": epc_data.get("walls-env-eff"),
+        "secondary_heating_description": epc_data.get("secondheat-description"),
+        "secondary_heating_energy_efficiency": epc_data.get("sheating-energy-eff"),
+        "secondary_heating_environmental_efficiency": epc_data.get("sheating-env-eff"),
+        "epc_roof_description": epc_data.get("roof-description"),
+        "roof_energy_efficiency": epc_data.get("roof-energy-eff"),
+        "roof_environmental_efficiency": epc_data.get("roof-env-eff"),
+        "main_heating_energy_efficiency": epc_data.get("mainheat-energy-eff"),
+        "main_heating_environmental_efficiency": epc_data.get("mainheat-env-eff"),
+        "lighting_description": epc_data.get("lighting-description"),
+        "lighting_energy_efficiency": epc_data.get("lighting-energy-eff"),
+        "lighting_environmental_efficiency": epc_data.get("lighting-env-eff"),
+        "main_fuel": epc_data.get("main-fuel"),
+        "floor_height": epc_data.get("floor-height"),
+        "mechanical_ventilation": epc_data.get("mechanical-ventilation"),
+        "construction_age_band": epc_data.get("construction-age-band"),
+        "tenure": epc_data.get("tenure"),
+        "uprn_source": epc_data.get("uprn-source"),
     }
+
+    if not exclude_pii:
+        row = {
+            **row,
+            "lmk_key": epc_data.get("lmk-key"),
+            "address_line_1": epc_data.get("address1"),
+            "address_line_2": epc_data.get("address2"),
+            "address_line_3": epc_data.get("address3"),
+        }
```

Comment:
We should have a record on the ticket (agreed with DESNZ) of what fields we're excluding. Can you open up that discussion please?

Reply:
can do